docs/developing.md

```diff
@@ -0,0 +1,43 @@
+# Developer's Guide
+
+## Getting the code
+
+Cloning this repository and change directory to it:
+```console
+$ go get -d github.com/deislabs/duffle/...
+$ cd $(go env GOPATH)/src/github.com/deislabs/duffle
+```
+
+## Building
+
+### Prerequisites
+You need:
+* A working Go 1.11.4 (or later) environment
```

Comment:
Any particular feature that requires 1.11.4?
I'm running 1.11.1, for example.

Reply:
@radu-matei I'm not aware of any. This just gives an impression about the starting point. It would be more relevant if the project was using modules, which it isn't.